{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_admin.py

```diff
@@ -47,3 +52,23 @@ def test_view_user(self, admin_client):
         url = reverse("admin:users_user_change", kwargs={"object_id": user.pk})
         response = admin_client.get(url)
         assert response.status_code == 200
+
+    @pytest.fixture
+    def force_allauth(self, settings):
+        settings.DJANGO_ADMIN_FORCE_ALLAUTH = True
+        # Reload the admin module to apply the setting change
+        import {{ cookiecutter.project_slug }}.users.admin as users_admin  # pylint: disable=import-outside-toplevel
+
+        try:
+            reload(users_admin)
+        except admin.sites.AlreadyRegistered:
+            pass
+
+    @pytest.mark.django_db
+    def test_allauth_login(self, rf, settings, force_allauth):  # pylint: disable=unused-argument
+        request = rf.get("/fake-url")
+        request.user = AnonymousUser()
+        response = admin.site.login(request)
+
+        # The `admin` login view should redirect to the `allauth` login view
+        assert response.status_code == 302
```

Comment:
Should we assert the `response.url` goes where we expect it to?

Reply:
Sure, I could add something like
```python
assert isinstance(response, HttpResponseRedirect) # for mypy
assert response.url == reverse(settings.LOGIN_URL) + "?next=" + request.path
```
(In which case I should add back the settings fixture). The `next` parameter has to be appended manually since it's not actually part of the defined url pattern.

Reply:
I implemented it with `assertRedirects` instead